core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -729,16 +743,16 @@ public static Builder builder(MetaData metaData) {
     }
 
     /** All known byte-sized cluster settings. */
-    public static final Set<String> CLUSTER_BYTES_SIZE_SETTINGS = ImmutableSet.of(
+    public static final Set<String> CLUSTER_BYTES_SIZE_SETTINGS = unmodifiableSet(newHashSet(
```

Comment:
I suppose the difference here is that its _our_ `Sets.newHashSet` and not Guava's. I don't have a strong opinion here other than that we shouldn't remove that helper method in this PR. If we want it gone it can go in another one.